samples/cpp/build_samples_msvc.bat

```diff
@@ -6,7 +6,7 @@
 @setlocal
 SETLOCAL EnableDelayedExpansion
 set "ROOT_DIR=%~dp0"
-FOR /F "delims=\" %%i IN ("%ROOT_DIR%") DO set SAMPLES_TYPE=%%~nxi
+FOR %%i IN ("%ROOT_DIR%\.") DO set SAMPLES_TYPE=%%~nxi
```

Comment:
why do we need it?
will it work if we call this script outside of current folder?

Reply:
That fixes exactly this use case. It used to work incorrectly

Reply:
but here in CI https://github.com/openvinotoolkit/openvino/blob/master/.ci/azure/windows.yml#L225-L229 we don't use current folder for running script. Why does it work on CI?

Reply:
It doesn't fail. But it uses current folder name instead of the folder in which the script is located

Reply:
ok, can we add such check (folder name) in azure CI?

Reply:
I don't know :) I'm going to abandon this PR for now since the major problem with arm64 isn't solved anyway.

Reply:
We can solve at least 2 other issues and skip only arm64 problem.